python/pyspark/resource/information.py

```diff
@@ -26,8 +26,10 @@ class ResourceInformation(object):
 
     One example is GPUs, where the addresses would be the indices of the GPUs
 
-    @param name the name of the resource
-    @param addresses an array of strings describing the addresses of the resource
+    :param name: the name of the resource
+    :param addresses: an array of strings describing the addresses of the resource
+
+    .. versionadded:: 3.0.0
```

Comment:
Will add a PR to backport this change specifically for branch-3.0. This PR itself should go to master only.

Reply:
Please make a PR to branch-3.0 for this part.